config.yaml

```diff
@@ -33,7 +33,7 @@ populate.commontypes: true
 # Authentication configuration
 # ----------------------------
 
-token.privatekey : > 
+token.privatekey : | 
```

Comment:
Thanks @kwk ,
Mentioned this in the description above - copying it here :)

~~Temporarily removed the keys from `config.yaml` because the format apparently is causing parsing errors.~~

**Use  _Literal style_ in config.yaml for rsa keys**

We were using [1] **>** for the multi-line RSA keys in config.yaml but **>** tends to eat up the newlines and eventually cause parsing errors. [2] So, that was replaced with **|** .

[1] https://github.com/almighty/almighty-core/blob/master/config.yaml#L36
[2] http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210